site/en/blog/optimizing-lcp-using-signed-exchanges/index.md

```diff
@@ -0,0 +1,459 @@
+{% endAside %}
+
+Another possible reason is the presence of the `Vary: Cookie` response header. If you serve different HTML to different users based on their cookie, then it wouldn't be correct for Google to cache and reuse an SXG of that HTML for multiple visitors.
+
+{% Aside %}
+In the future, you will be able to designate an SXG as only for new visitors, using `Vary: Cookie`. This is [implemented in Chromium](https://crbug.com/1250532), and will be enabled in Google Search once that has rolled out to enough people.
+{% endAside %}
+
+Alternatively to the Chrome extension, you can use a tool like `curl`:
+
+```bash
+curl -siH "Accept: application/signed-exchange;v=b3" $URL | less
+```
+
+or [`dump-signedexchange`](https://github.com/WICG/webpackage/blob/main/go/signedexchange/README.md#dump-a-signed-exchange-file):
+
+```bash
+dump-signedexchange -verify -uri $URL
+```
+
+### Indexing
+
+Make sure your SXGs are successfully [indexed](https://support.google.com/webmasters/answer/7645831) by Google Search. Test if Google Search is linking to webpkgcache.com's copy of your page, by looking in DevTools. Since Google Search currently only supports SXG for Android OSes, you may need to enable mobile-emulation mode and select an Android device:
+
+{% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/Qu2qWl2LES3N8tiqnskP.png",
+       alt="Google Search results with DevTools showing an <a> that points to webpkgcache.com",
+       width="800", height="635" %}
+
+If Google Search thinks the user is likely to click on the result, it will also prefetch it:
+
+{% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/uhchyec7uo4rgcQsteaE.png",
+       alt="Google Search results with DevTools showing a <link rel=prefetch> for webpkgcache.com",
+       width="800", height="147" %}
+
+The `<link>` tag instructs the browser to download the SXG into its prefetch cache. When the user clicks on the `<a>` tag, the browser will use that cached SXG to render the page.
+
+You can also see evidence of this by going to the Network tab in DevTools and searching for URLs containing `webpkgcache`.
+
+If this is happening, this means that Google Search indexing of the signed exchange is working. You can skip forward to the [Ingestion](#ingestion) section.
+
+Otherwise, it could be that Google hasn't recrawled the page yet since you enabled SXG. Try the Search Console URL Inspection Tool:
```

Comment:
```suggestion
Otherwise, it could be that Google hasn't recrawled the page yet since you enabled SXG. Try the [Google Search Console](https://search.google.com/search-console/about) URL Inspection Tool:
```